assembly/assembly-main/src/assembly/bin/docker.sh

```diff
@@ -121,10 +121,6 @@ init() {
     export CHE_WORKSPACE_STORAGE_CREATE_FOLDERS=false
   fi
 
-  # Ensure that the user "user" has permissions for CHE_HOME and CHE_DATA
-  sudo chown -R user:user ${CHE_HOME}
-  sudo chown -R user:user ${CHE_DATA}
```

Comment:
@l0rd also I think workspaces only need access to /projects which is done there
https://github.com/eclipse/che/blob/5fbaf3d9ac45344b5596742b27d9335652d3d0ca/wsmaster/che-core-api-agent/src/main/resources/agents/scripts/org.eclipse.che.ws-agent.script.sh#L38